### DISCUSSION TITLE
Question about bond

### DISCUSSION THREAD

**Discussion by UNSCs117** [lonqie/SchaleDB#93] [Questions] [answered]
I offen use schale.gg as bond rank recorder, but when I was forcibly transferred to schaledb.com today, I fond that I can't edit bond data for any student, neither "skills - calculation" nor "profiles".
If possible, I want be able to adjust the student's Bond Rank data directly within "Profiles", maybe that can be a new function. Thanks!

**Comment by lonqie** [comment, marked as answer]
Hi, bond values are changed by typing the value into the fields shown.

![image](https://github.com/user-attachments/assets/2de93bf8-c271-488e-8976-12eac950dd60)

The bond stats shown on the profile 'tab' are a separate slider and not tied to the saved values for bond ranks. This is how it worked back on the old version and there is no plan to change that behaviour.

**Reply by UNSCs117** [reply]
That's ... quiet amazing, I didn't expect this😂
I thought a slider will be used😂
